launcher/resources/multimc/index.theme

```diff
@@ -1,5 +1,5 @@
 [Icon Theme]
-Name=multimc
+Name=Legacy
```

Comment:
I'm sure this makes sense, but... are these translatable or do we want them to be? (the original down in ThemeCustomizationWidget was translated all)
I suppose its fine for us to not translate, but idk what other ppl have to say or its even possible when going this route

Reply:
oh. well previously the name field was unused and having a hardcoded map is not nice

Reply:
its probably fine this way, its just a thing i noticed while reading through the code

Reply:
A lot of names should be translated, but on the other hand i don't think it's a big deal considering you can see what the icons look like easily :P